### PR TITLE
Allow overriding the log level from the config file

### PR DESCRIPTION
**Pull request by philippw**
You now can set the log level from the stryker4s.conf file using the `log-level` property

Accepted values are ERROR, WARN, INFO, DEBUG, TRACE, ALL, OFF

Fixes bullet point 1 in #41

### CODE REVIEW

File: util/src/test/scala/stryker4s/scalatest/LogMatchers.scala [hunk 1]
```diff
@@ -33,12 +33,14 @@ trait LogMatchers {
   def loggedAsInfo = new LogMatcherWithLevel(Level.INFO)
   def loggedAsWarning = new LogMatcherWithLevel(Level.WARN)
   def loggedAsError = new LogMatcherWithLevel(Level.ERROR)
+  def notLoggedAtAll = new LogMatcherWithLevel(Level.OFF)
```
Comment (hugo-vrijswijk) [inline util/src/test/scala/stryker4s/scalatest/LogMatchers.scala]: How can a message be logged with log level OFF?

File: core/src/main/scala/stryker4s/run/process/ProcessRunner.scala [hunk 2]
```diff
@@ -11,8 +11,8 @@ import scala.util.Try
 
 trait ProcessRunner extends Logging {
   def apply(command: Command, workingDir: File, envVar: (String, String)): Try[Int] = {
-    val mutantProcess = Process(command.command + " " + command.args, workingDir.toJava, envVar)
-      .run(ProcessLogger(s => debug(s)))
+    val mutantProcess = Process(Seq(command.command, command.args), workingDir.toJava, envVar)
```
Comment (hugo-vrijswijk) [inline core/src/main/scala/stryker4s/run/process/ProcessRunner.scala]: Why is this changed? Wrong merge?

File: core/src/main/scala/stryker4s/config/Config.scala [hunk 3]
```diff
@@ -3,15 +3,22 @@ package stryker4s.config
...
 import better.files._
+import ch.qos.logback.classic.Level
 import com.typesafe.config.ConfigRenderOptions
 import pureconfig.ConfigWriter
 
 case class Config(files: Seq[String] = Seq("**/main/scala/**/*.scala"),
                   baseDir: File = File.currentWorkingDirectory,
-                  testRunner: TestRunner = CommandRunner("sbt", "test")) {
+                  testRunner: TestRunner = CommandRunner("sbt", "test"),
+                  logLevel: Level = Level.DEBUG) {
```
Comment (hugo-vrijswijk) [inline core/src/main/scala/stryker4s/config/Config.scala]: Can we change the default log level to INFO? Makes more sense to me, and it's also what Stryker does.

Comment (philippw) [reply core/src/main/scala/stryker4s/config/Config.scala]: Yes, I'll do that.
